samples/boards/nrf/ieee802154/802154_rpmsg/README.rst

```diff
@@ -0,0 +1,32 @@
+.. _nrf-ieee802154-rpmsg-sample:
+
+nRF IEEE 802.15.4: Serialization RPMsg
+######################################
+
+Overview
+********
+
+This sample exposes IEEE 802.15.4 radio driver support
+to another device or CPU using RPMsg transport which is
+a part of `OpenAMP <https://github.com/OpenAMP/open-amp/>`__.
+
+Requirements
+************
+
+* A board with :ref:`ipm_api` driver and nRF IEEE 802.15.4 radio driver support
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/boards/nrf/ieee802154/802154_rpmsg`
+in the Zephyr tree.
+
+To use this application, you need a board with Nordic 802.15.4 radio and IPM drivers.
```

Comment:
Would be good to highlight the nRF53 here.

Reply:
Like above: replaced conditions list with the name of the chip meeting all conditions. I think it's clearer this way.